### PR TITLE
Add mappings for kucoin assets

### PR DESCRIPTION
**Pull request by moxw**

### CODE REVIEW

File: updates/location_asset_mappings/v2.json [hunk 1]
```diff
@@ -90,8 +90,219 @@
...
       ]
     },
-    "updates": []
+    "updates": [
+      {
+        "asset": "eip155:56/erc20:0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82",
+        "location": null,
+        "location_symbol": "CAKE"
+      }
+    ]
```
[the remark this answers is not in the document]
Comment (moxw) [reply updates/location_asset_mappings/v2.json]: CAKE was already present in v1.json, removed

File: updates/location_asset_mappings/v2.json [hunk 2]
```diff
@@ -90,8 +90,104 @@
...
+            "asset": "eip155:137/erc20:0x5F2F8818002dc64753daeDF4A6CB2CcB757CD220",
+            "location": "kucoin",
+            "location_symbol": "WSDM"
         }
       ]
     },
-    "updates": []
+    "updates": [
+      {
+        "asset": "eip155:56/erc20:0xeC5483804e637D45cDe22Fa0869656B64b5AB1ab",
```
Comment (OjusWiZard) [inline updates/location_asset_mappings/v2.json:188]: this looks like on a wrong chain, should be mainnet right?

File: updates/location_asset_mappings/v2.json [hunk 3]
```diff
@@ -90,8 +90,104 @@
...
+        },
+        {
+            "asset": "eip155:137/erc20:0x5F2F8818002dc64753daeDF4A6CB2CcB757CD220",
+            "location": "kucoin",
+            "location_symbol": "WSDM"
+        },
+        {
+            "asset": "eip155:137/erc20:0x5F2F8818002dc64753daeDF4A6CB2CcB757CD220",
+            "location": "kucoin",
+            "location_symbol": "WSDM"
```
Comment (OjusWiZard) [inline updates/location_asset_mappings/v2.json:182]: this is same as above

Comment (moxw) [reply updates/location_asset_mappings/v2.json:182]: removed

File: updates/location_asset_mappings/v2.json (hunk 2 above)
Comment (moxw) [reply updates/location_asset_mappings/v2.json:188]: right fixed

File: updates/location_asset_mappings/v2.json [hunk 4]
```diff
@@ -90,8 +90,99 @@
...
       ]
     },
-    "updates": []
+    "updates": [
+      {
+        "asset": "eip155:1/erc20:0xeC5483804e637D45cDe22Fa0869656B64b5AB1ab",
+        "location": "kucoin",
+        "location_symbol": "ACE"
+      }
+    ]
```
Comment (OjusWiZard) [inline updates/location_asset_mappings/v2.json:187]: this updates section is outside of `location_asset_mappings` key, should be inside it

**Review by OjusWiZard — APPROVED** [review]
lgtm

**Pull request merged**